### ISSUE TITLE
Accommodate for longer words for app status and app version on side navigation

### ISSUE THREAD

**Issue by GCHQ-Developer-847** [mi6/ic-ui-kit#550]
## Summary
Make the app status and app version wrap onto multiple lines instead of being truncated. Remove max-width on app status tag and allow the tag and app version to use up the available width within the side navigation. 

## 💬 Description
The app status and app version on the side navigation are currently truncated if their values are too long to be fully displayed. Having this behaviour alongside a tooltip to show the whole value would be an accessibility issue as there is no way for a keyboard user to see the whole values (without making both elements focusable when they shouldn't be - as they are not interactive).

To solve this, the app status and app version should wrap onto multiple lines instead of being truncated.

There is also a max-width of 6rem on the app status tag. To make the best use of space within the bottom section of the side navigation, the max-width should be removed and the app status and app version should be allowed to take up the whole available width (between the expand icon - or left side of the side navigation if the static variant is being used - and the right side). 

See the Figma file "Side navigation app status and version number wrapping" in "ICDS - Design Contributions". The status and version each grow to take up the remaining width and then wrap onto multiple lines. The height of the bottom app info section should always be the same when the side navigation is collapsed.

Long status:
![image](https://user-images.githubusercontent.com/111882109/231526464-8a77fb21-2445-485f-9fde-90e9dc1aee18.png)

Long version:
![image](https://user-images.githubusercontent.com/111882109/231526532-17d78925-5b77-41be-8478-b012f97e4885.png)

Long version and long status (they take up equal width):
![image](https://user-images.githubusercontent.com/111882109/231526761-7227c07d-59d1-4d8a-aac7-a7d961dc192a.png)

**Note:** the gap between the version and the right edge of side navigation is currently 1rem, whereas it is 0.5rem in the designs. This will also need updating (and moreover will help increase the available space for the status tag further).

-----

**To summarise - changes required in this ticket**:
- Allow the app status text and app version text to wrap onto multiple lines instead of being truncated
- Remove max-width on app status tag - don't have a max-width on either the status or version, and instead let them take up the whole available width
- Change gap on the right of the app version to 0.5rem

-----

It might be worth looking at CSS flex options ([A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)) - it looks like one of these might be what we're looking for (maybe flex-grow / flex-shrink?).

## 💰 User value
This will provide more space for the app status and app version and avoid the accessibility issues that come with using truncation and tooltips.

**Comment by GCHQ-Developer-847** [comment]
Related: ticket for displaying tooltip when text is truncated - [#551](https://github.com/mi6/ic-ui-kit/issues/551)
